transformers/storage/repository.go

```diff
@@ -219,6 +220,19 @@ func (repository *MakerStorageRepository) GetFlipBidIds(contractAddress string)
 	return bidIds, err
 }
 
+func (repository *MakerStorageRepository) GetPotPieUsers() ([]string, error) {
+	var userAddresses []string
+	err := repository.db.Select(&userAddresses, `
+		SELECT addresses.address
+		FROM maker.pot_join
+		    LEFT JOIN public.addresses ON pot_join.msg_sender = addresses.id
+		UNION
+		SELECT addresses.address
+		FROM maker.pot_exit
+		    LEFT JOIN public.addresses ON pot_exit.msg_sender = addresses.id`)
```

Comment:
Seems legit and follows the pattern we've used, but there should never be an address exiting without a prior join - right? Totally fine with leaving things as is to avoid errors from missed events, just curious

Reply:
Yeah, totally! I hadn't really thought about that. Hopefully one day we'll have a node so reliable that we won't need to be redundant :)